src/fontsize/fontsizeediting.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * @license Copyright (c) 2003-2018, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/**
+ * @module font/fontsize/fontsizeediting
+ */
+
+import Plugin from '@ckeditor/ckeditor5-core/src/plugin';
+import { attributeToElement } from '@ckeditor/ckeditor5-engine/src/conversion/two-way-converters';
+
+import FontSizeCommand from './fontsizecommand';
+import { normalizeOptions } from './utils';
+
+const FONT_SIZE = 'fontSize';
+
+/**
+ * The Font Size Editing feature.
```

Comment:
What does it do? Since we break features into pieces we must explain what each piece is responsible for if developers are expected to use them.